src/components/Footer.jsx

```diff
@@ -57,9 +85,43 @@ class SiteFooter extends React.Component {
             <img
               style={{ maxHeight: 45 }}
               src={logo || config.LOGO_TRADEMARK_URL}
-              alt={intl.formatMessage(messages['footer.logo.altText'])}
+              alt={getConfig().FOOTER_LOGO_ALT_TEXT || intl.formatMessage(messages['footer.logo.altText'])}
             />
           </a>
+          )}
+          <div className="copyright-col">
+            {getConfig().TRADEMARK_TEXT
+            && (
+            <div className="text-gray-500 small">
```

Comment:
Shouldn't we be using more semantic class names?  Specially since then it's easier for theming, later.